extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -64,14 +64,14 @@ define('ext.wikia.adEngine.adContext', [
 				geo.isProperGeo(instantGlobals.wgAdDriverSourcePointDetectionMobileCountries));
 		}
 
-		// SourcePoint integration
-		if (context.opts.sourcePointDetection && context.opts.sourcePointUrl) {
-			context.opts.sourcePoint = isUrlParamSet('sourcepoint') ||
-				geo.isProperGeo(instantGlobals.wgAdDriverSourcePointCountries);
+		// SourcePoint recovery integration
```

Comment:
I'm afraid we have to keep the old conditions for a while too since JS and AdContext can be desync'ed.